pd/src/mempool.rs

```diff
@@ -32,29 +32,22 @@ use tokio::sync::{watch, Mutex as AsyncMutex};
 use tower_abci::BoxError;
 use tracing::Instrument;
 
-use crate::{state, verify::StatelessTransactionExt, RequestExt};
+use crate::RequestExt;
 
 #[derive(Clone)]
 pub struct OldMempool {
     // sidecar
     new_mempool: Mempool,
     // old code
     nullifiers: Arc<AsyncMutex<BTreeSet<Nullifier>>>,
```

Comment:
The whole `OldMempool` struct can be deleted, since we don't use it at all any more.